docker/linea-sepolia/docker-compose.yml

```diff
@@ -51,24 +58,28 @@ services:
       - ./genesis.json:/genesis.json:ro
       - linea-sepolia-geth:/root/.ethereum
 
-  # Besu node
+  # Besu node (Execution Layer for Maru)
   besu-node:
-    hostname: besu-node
-    container_name: besu-node
-    image: consensys/linea-besu:${SEQUENCER_TAG:-linea-delivery-17}
+    hostname: linea-besu
+    container_name: linea-besu
+    image: hyperledger/besu:25.8.0 #25.8 +
```

Comment:
`linea-besu` is outtdated, but I think it's better to use `linea-besu-package` here now, because that's the recommended way